pom.xml

```
@@ -185,7 +183,7 @@
 							<goal>process</goal>
 						</goals>
 						<configuration>
-							<outputDirectory>target/generated-sources</outputDirectory>
+							<outputDirectory>src/generated-sources</outputDirectory>
```

```suggestion
							<outputDirectory>target/generated-sources</outputDirectory>
```
Q클래스 github에 올리지 마시죠 ㅠ 
회사마다 다르긴 하다는데, 엔티티 수정 되면 컴파일 되면서 Q클도 바뀌어 가지고.
.gitignore에 target 등록되어 잇으니
target으로 가시죠